src/main/java/com/ibm/cics/bundle/parts/AbstractJavaBundlePart.java

```diff
@@ -0,0 +1,93 @@
+package com.ibm.cics.bundle.parts;
+
+/*-
+ * #%L
+ * CICS Bundle Maven Plugin
+ * %%
+ * Copyright (C) 2019 IBM Corp.
+ * %%
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * #L%
+ */
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.function.Consumer;
+
+import javax.xml.transform.TransformerException;
+
+import org.apache.commons.io.FileUtils;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import com.ibm.cics.bundle.parts.BundlePublisher.PublishException;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public abstract class AbstractJavaBundlePart extends BundlePart {
+
+	private final String jvmServer;
+	private final File bin;
+	private final String binExtension;
+	private final String symbolicName;
+	
+	public AbstractJavaBundlePart(
+			String name,
+			BundlePartType type,
+			String symbolicName,
+			String jvmServer,
+			File bin,
+			String binExtension) {
+		super(name, type);
+		//TODO: make sure Maven plugin sets name to name-version
```

Comment:
Do you still need to address these TODOs?

Reply:
Confused, I think I already removed it? :s

Reply:
Try refreshing the PR page!

Reply:
could be because I was viewing by commit! Will go through by files instead!